docs/lang_spec/Cadence Language Specification.md

```diff
@@ -0,0 +1,5798 @@
+
+# Cadence Language Specification
+
+**THIS IS THE START OF A SPECIFICATION**
+**: It is incomplete and may contain many errors.**
+
+## I. Introduction
+
+### 1. Purpose of the Specification
+The purpose of this specification is to provide a formal and comprehensive definition of the Cadence programming language. Cadence is specifically designed for writing smart contracts and decentralized applications (dApps) on the Flow blockchain. This document serves as an authoritative guide for the language's syntax, semantics, type system, and execution model, ensuring consistent implementation and usage across the Flow ecosystem.
```

Comment:
Cadence is not Flow-specific, but a general smart contract language

```suggestion
The purpose of this specification is to provide a formal and comprehensive definition of the Cadence programming language. Cadence is specifically designed for writing smart contracts and decentralized applications (dApps) on the blockchains. This document serves as an authoritative guide for the language's syntax, semantics, type system, and execution model, ensuring consistent implementations.
```

Please also correct this in the remainder of the section

Reply:
Updated.